internal/collections.go

```diff
@@ -0,0 +1,169 @@
+/*******************************************************************************
+ * Copyright 2019 VMWare Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *******************************************************************************/
+package internal
+
+import (
+	"github.com/globalsign/mgo"
+	"github.com/globalsign/mgo/bson"
+
+	"github.com/edgexfoundry/docker-edgex-mongo/internal/pkg"
+)
+
+func creatAdminCollections(db *mgo.Database) {
+	_, err := db.C("system.users").RemoveAll(nil)
+	if err != nil {
+		pkg.LoggingClient.Error("Error during execution: " + err.Error())
+	}
+
```

Comment:
In this function I strongly recommend removing the remaining logic which removes default version information and then re-writes it with an incorrect version value. We shouldn't be managing this. The Mongo version we're using is 4 and this logic sets it to 3.

I have connected to a default `mongo:4.0-xenial` container and queried this information
```
> show dbs
admin   0.000GB
config  0.000GB
local   0.000GB
> use admin
switched to db admin
> show collections
system.version
> db.system.version.find()
{ "_id" : "featureCompatibilityVersion", "version" : "4.0" }
``` 
We should leave it as such.

Reply:
removed